app/src/main/java/org/apache/taverna/mobile/ui/workflow/WorkflowPresenter.java

```diff
@@ -111,33 +111,36 @@ public void onError(Throwable e) {
                     }
 
                     @Override
-                    public void onNext(String s) {
-                        getMvpView().performSearch(s);
-                        if (!TextUtils.isEmpty(s)) {
-                            searchWorkflow(1, s);
-                        }
+                    public void onComplete() {
+
                     }
-                });
-        mSubscriptions.add(mSearchViewSubscription);
+                }));
+        compositeDisposable.add(compositeDisposable);
```

Comment:
As we are using `CompositeDisposable`, Please remove this line. this line is making no sense.